src/cdk/v2/destinations/http/procWorkflow.yaml

```diff
@@ -38,16 +38,18 @@ steps:
       const additionalConfigHeaders = $.getCustomMappings(.message, .destination.Config.headers);   
       $.context.headers = {
         ...configAuthHeaders,
-        ...additionalConfigHeaders
+        ...additionalConfigHeaders,
+        "Content-Type": $.context.format === "JSON" ? "application/json" : "application/xml",
```

Comment:
write a function to convert to context.format to correct context type

Reply:
Done